client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ContactsBank/ContactsBankForm.tsx

```diff
@@ -1,19 +1,20 @@
-import React, {useContext} from 'react';
+import React, { useContext, useState } from 'react';
 import { Typography, Grid, Paper } from '@material-ui/core';
 
 import Contact from 'models/Contact';
 import {contactBankContext} from 'commons/Contexts/ContactBankContext';
 
 import useStyles from './contactsBankFormStyles';
 import ContactsBankTable from './ContactsBankTable/ContactsBankTable';
+import ContactsBankSearchBar from './ContactsBankSearchBar/ContactsBankSearchBar';
 
 const headline = 'בנק מגעים:';
 
 const ContactsBankForm = (props: Props) => {
     const { existingPersons } = props;
     const classes = useStyles(); 
     const {contactBank} = useContext(contactBankContext);
-    
+    const [query, setQuery] = useState<string>("");
```

Comment:
change to single quotes

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1363/commits/84f1f190092001b223eb1e4c1b6419d0f91894c5)